### PR TITLE
Created Custom Turtle post without the assignment in there yet

### PR DESCRIPTION
**Pull request by EAkita**

### CODE REVIEW

File: _posts/eakita/2020-09-02-eakita-custom-turtle.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+author: eakita
+title: "EAkita's Custom Turtle
```
Comment (eah13) [inline _posts/eakita/2020-09-02-eakita-custom-turtle.md]: Missing the second `"` here.

**Review by eah13 — CHANGES_REQUESTED** [review]
You're missing a `"` but everything else is technically correct.

Some of your messages to humans are misleading, though. When you edited the text to embed the trinket it would've been nice to change the post text to something like "Here's my custom turtle:".

And the pull request title says the assignment isn't in there yet but it is! Be nice to your humans and try to keep the title and description of your pull requests updated alongside the code.

Since you'll already be editing the post, consider addressing these as well.

File: _posts/eakita/2020-09-02-eakita-custom-turtle.md (hunk 1 above)
Comment (EAkita) [reply _posts/eakita/2020-09-02-eakita-custom-turtle.md]: @eah13 Thanks! just made the change

**Comment by EAkita** [comment]
@eah13 made the change. Kindly review when you get a chance

**Review by eah13 — APPROVED** [review]
Code looks good! Your messages to humans on this pull request (title, description) could've been a little more descriptive though.

**Pull request merged**